test/run_test.py

```diff
@@ -1402,7 +1402,6 @@ def parse_args():
         )
         and get_pr_number() is not None
         and not strtobool(os.environ.get("NO_TD", "False"))
-        and not TEST_WITH_ROCM
         and not IS_MACOS
         and "xpu" not in BUILD_ENVIRONMENT
         and "onnx" not in BUILD_ENVIRONMENT
```

Comment:
GH won't let me comment on it, but the line right below this one might also need a change, depending on where you want to run TD for rocm

To check if tests are not being run, search for "excluded" in the test logs.  If you click show additional test info on HUD, there's also a td section that should say